index.html

```diff
@@ -51,6 +51,114 @@ <h1 class="my-name" >I'm Omotoyosi. <br>
+                <li> <button class="stack-button"><a href="http://"> HTML</a></button></li>
+                <li><button class="stack-button"><a href="http://"> CSS</a></button></li>
+                <li><button class="stack-button"><a href="http://">JavaScript</a></button></li>
+            </ul>
+            <button class="see-proj" ><a class="project-link" href="">See project</a></button>
+        </div>
+        <div class="item works-3">
+            <img src="images/works-3.svg" alt="template-image" srcset="">
+            <p class="title-1">Tonic</p>
+            <ul class="track">
+                <li class="live">CANOPY</li>
+                <li><img src="images/Counter.svg" alt="counter" srcset=""></li>
+                <li>Back End Dev</li>
+                <li><img src="images/Counter.svg" alt="" srcset=""></li>
+                <li>2015</li>
+            </ul>
+            <p class="some-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="stacks">
+                <li> <button class="stack-button"><a href="http://"> HTML</a></button></li>
+                <li><button class="stack-button"><a href="http://"> CSS</a></button></li>
+                <li><button class="stack-button"><a href="http://">JavaScript</a></button></li>
+            </ul>
+            <button class="see-proj" ><a class="project-link" href="">See project</a></button>
+        </div>
+        <div class="item works-4">
+            <img src="images/works-4.svg" alt="template-image" srcset="">
+            <p class="title-1">Multi-Post <br> Stories</p>
+            <ul class="track">
+                <li class="live">CANOPY</li>
+                <li><img src="images/Counter.svg" alt="counter" srcset=""></li>
+                <li>Back End Dev</li>
+                <li><img src="images/Counter.svg" alt="" srcset=""></li>
+                <li>2015</li>
+            </ul>
+            <p class="some-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="stacks">
+                <li> <button class="stack-button"><a href="http://"> HTML</a></button></li>
+                <li><button class="stack-button"><a href="http://"> CSS</a></button></li>
+                <li><button class="stack-button"><a href="http://">JavaScript</a></button></li>
+            </ul>
+            <button class="see-proj" ><a class="project-link" href="">See project</a></button>
+        </div>
+    </section>
+    <section class="third-section">
+        <div class="content">
+            <p class="about">About Myself</p>
+            <p class="about-info">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+
+            <p class="connect">
+                LET’S CONNECT
+            </p>
```

Comment:
- [ ] The title of this section and the 'LET'S CONNECT' title would be better wrapped with some `hx` tag. Please change the tags to a proper `h2` or other `hx`(the x should be a number, not the letter 'x') tag that follows the structure of the page.